### PR TITLE
Update Insteon documentation

### PR DESCRIPTION
**Pull request by teharris1**
## Proposed change

Update the Insteon documentation to align with changes introduced in 2024.5.0

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/insteon.markdown [hunk 1]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+The first time autodiscovery runs, **any device that is already linked to the modem** will be identified. This process may require up to 60 seconds per device. Subsequent startups will occur much quicker using cached device information. If a device is not recognized during autodiscovery, trigger the device, such as toggling a button, to force the device to send a message to the modem. The device will then be discovered. You may need to trigger the device a few times. If the device is still not identified, relink the device to the modem following the linking instructions using the [Insteon configuration panel](#insteon-configuration-panel).
 
-## Insteon Panel
+## Insteon Configuration Panel
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
## Insteon configuration panel
```

File: source/_integrations/insteon.markdown [hunk 2]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+In order for any two Insteon devices to talk with one another, they must be linked. For an overview of device linking, please read the Insteon page on [understanding linking]. To add an Insteon device:
+
+1. Open the [Insteon configuration panel](#insteon-configuration-panel).
+2. Press the **Add device** button on the bottom right corner of the screen.
+3. Fill in the **Add device** dialog using the following options:
+
+- **Device address**: Enter the Insteon address of the device, such as `1A.2B.3C`, to link a specific device remotely. Not all devices support remote linking. If a device does not support remote linking, press the `Set` button on the device to put the device in linking mode.
+- **Link multiple**: To add multiple devices in a single session, select the **Add multiple** checkbox.
+
+### Managing Insteon Scenes
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
### Managing Insteon scenes
```

File: source/_integrations/insteon.markdown [hunk 3]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+
+Insteon scenes can be created, changed or deleted using the **Scenes** tab of the [Insteon configuration panel](#insteon-configuration-panel). To trigger an Insteon scene see [Triggering Insteon Scenes](#triggering-insteon-scenes) below.
+
+### Device Properties
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
### Device properties
```

File: source/_integrations/insteon.markdown [hunk 4]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
 [understanding linking]: https://www.insteon.com/support-knowledgebase/2015/1/28/understanding-linking
 
-## INSTEON Scenes
+### Utilities
+
+- **Change the modem connection**: Reconfigure the modem connection information such as USB port or Hub IP address.
+- **Configure device overrides**: Add or remove device overrides. See [Device Overrides](#device-overrides) below.
+- **Delete device**: Delete an Insteon device from the network using the device's Insteon address.
+
+## Triggering Insteon Scenes
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
## Triggering Insteon scenes
```

File: source/_integrations/insteon.markdown [hunk 5]
```diff
@@ -160,7 +191,7 @@ The following services are available:
 - **insteon.delete_all_link**: Tells the Insteon Modem (IM) to remove an All-Link record from the All-Link Database of the IM and a device. Once the IM is set to delete the link, press the SET button on the corresponding device to complete the process.
 - **insteon.load_all_link_database**: Load the All-Link Database for a device. WARNING - Loading a device All-Link database may take a LONG time and may need to be repeated to obtain all records.
 - **insteon.print_all_link_database**: Print the All-Link Database for a device. Requires that the All-Link Database is loaded first.
-- **insteon.print_im_all_link_database**: Print the All-Link Database for the INSTEON Modem (IM).
+- **insteon.print_im_all_link_database**: Print the All-Link Database for the Insteon Modem (IM).
 - **insteon.add_default_links**: Add a set of default links between the modem and the device to facilitate proper communication between them.
 
 ## Device Overrides
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
## Device overrides
```

File: source/_integrations/insteon.markdown [hunk 6]
```diff
@@ -172,7 +203,7 @@ There are two primary uses for the **device override** feature:
 - Devices that do not respond during autodiscovery. This is common for battery operated devices. Before using a device override, please trigger the device a few times and it will likely be discovered by Home Assistant.
 - Devices that have not been fully developed. This allows an unknown device to be mapped to a device that operates similarly to another device.
 
-Device overrides can be set up using the integrations page inside the configuration panel.
+Device overrides can be set up using the [Insteon configuration panel](#insteon-configuration-panel).
 
 ## Recovering After Factory Resetting The Hub
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
## Recovering after factory resetting the hub
```

**Review by c0ffeeca7 — COMMENTED** [review]
I realize some of these comments are outside the scope of this PR. But we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headings.

File: source/_integrations/insteon.markdown [hunk 7]
```diff
@@ -29,9 +29,9 @@ ha_dhcp: true
 
 <p class='note warning'>The Insteon apps (Director or Insteon for Hub) are a paid service utilizing the Insteon cloud to control an Insteon Hub. Home Assistant does not require the use of the Insteon app but can operate in conjunction with the app if desired.</p>
 
-This integration adds support for integrating your INSTEON network with Home Assistant. It has been tested with all USB and serial PowerLinc Modems (PLM) including [2413U], [2448A7], [2413S] and [2412S] models. It has also been tested to work with the [2242] and [2245] Hubs.
+This integration adds support for integrating your Insteon network with Home Assistant. It has been tested with all USB and serial PowerLinc Modems (PLM) including [2413U], [2448A7], [2413S] and [2412S] models. It has also been tested to work with the [2242] and [2245] Hubs.
 
-_If you have factory reset your device please see the instructions <a href="#recovering-after-factory-resetting-the-hub">Recovering After Factory Resetting The Hub</a> for how to proceed._
+_If you have factory reset your device please see the instructions [Recovering After Factory Resetting The Hub](#recovering-after-factory-resetting-the-hub) for how to proceed._
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
_If you have factory reset your device, please see the instructions [Recovering After Factory Resetting The Hub](#recovering-after-factory-resetting-the-hub) for how to proceed._
```

File: source/_integrations/insteon.markdown [hunk 8]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+The first time autodiscovery runs, **any device that is already linked to the modem** will be identified. This process may require up to 60 seconds per device. Subsequent startups will occur much quicker using cached device information. If a device is not recognized during autodiscovery, trigger the device, such as toggling a button, to force the device to send a message to the modem. The device will then be discovered. You may need to trigger the device a few times. If the device is still not identified, relink the device to the modem following the linking instructions using the [Insteon configuration panel](#insteon-configuration-panel).
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
The first time autodiscovery runs, **any device that is already linked to the modem** will be identified. This process may require up to 60 seconds per device. Subsequent startups will occur much quicker using cached device information. If a device is not recognized during autodiscovery, trigger the device&mdash;for example, by toggling a button&mdash;to force the device to send a message to the modem. The device will then be discovered. You may need to trigger the device a few times. If the device is still not identified, relink the device to the modem following the linking instructions using the [Insteon configuration panel](#insteon-configuration-panel).
```

File: source/_integrations/insteon.markdown [hunk 9]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+
+### Device properties
+
+Insteon device properties, such as the LED brightness, can be managed using the Insteon configuration panel. To see the available properties of a device, select the device from the list of devices in the Insteon configuration panel. This will display the list of available properties for the specific device on the **Properties** tab. Each device type will have a different set of properties and not all devices have properties.
+
+- **Read device properties**:  Use the **Load from device** menu option to read the properties from the device.
+- **Change device properties**: Select the specific property from the list of properties. Use the dialog box to edit the property value. This does not write the change to the device.
+- **Write property changes**: Use the **Write to device** menu option to save the property changes to the device.
+- **Undo changes**: Property changes can be undone before they are written to the device by selecting the **Undo changes** menu option.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
- **Undo changes**: Undoes property changes before they are written to the device.
```

File: source/_integrations/insteon.markdown [hunk 10]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+
+Insteon scenes can be created, changed or deleted using the **Scenes** tab of the [Insteon configuration panel](#insteon-configuration-panel). To trigger an Insteon scene see [Triggering Insteon Scenes](#triggering-insteon-scenes) below.
+
+### Device properties
+
+Insteon device properties, such as the LED brightness, can be managed using the Insteon configuration panel. To see the available properties of a device, select the device from the list of devices in the Insteon configuration panel. This will display the list of available properties for the specific device on the **Properties** tab. Each device type will have a different set of properties and not all devices have properties.
+
+- **Read device properties**:  Use the **Load from device** menu option to read the properties from the device.
+- **Change device properties**: Select the specific property from the list of properties. Use the dialog box to edit the property value. This does not write the change to the device.
+- **Write property changes**: Use the **Write to device** menu option to save the property changes to the device.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown:89]:
```suggestion
- **Write property changes**: Saves the property changes to the device.
```

File: source/_integrations/insteon.markdown [hunk 11]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+### Managing Insteon scenes
+
+Insteon scenes can be created, changed or deleted using the **Scenes** tab of the [Insteon configuration panel](#insteon-configuration-panel). To trigger an Insteon scene see [Triggering Insteon Scenes](#triggering-insteon-scenes) below.
+
+### Device properties
+
+Insteon device properties, such as the LED brightness, can be managed using the Insteon configuration panel. To see the available properties of a device, select the device from the list of devices in the Insteon configuration panel. This will display the list of available properties for the specific device on the **Properties** tab. Each device type will have a different set of properties and not all devices have properties.
+
+- **Read device properties**:  Use the **Load from device** menu option to read the properties from the device.
+- **Change device properties**: Select the specific property from the list of properties. Use the dialog box to edit the property value. This does not write the change to the device.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
- **Change device properties**: Allows you to select a specific property from a list of properties and edit the property values. This does not write the change to the device.
```

File: source/_integrations/insteon.markdown [hunk 12]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+
+### Managing Insteon scenes
+
+Insteon scenes can be created, changed or deleted using the **Scenes** tab of the [Insteon configuration panel](#insteon-configuration-panel). To trigger an Insteon scene see [Triggering Insteon Scenes](#triggering-insteon-scenes) below.
+
+### Device properties
+
+Insteon device properties, such as the LED brightness, can be managed using the Insteon configuration panel. To see the available properties of a device, select the device from the list of devices in the Insteon configuration panel. This will display the list of available properties for the specific device on the **Properties** tab. Each device type will have a different set of properties and not all devices have properties.
+
+- **Read device properties**:  Use the **Load from device** menu option to read the properties from the device.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
- **Read device properties**:  Reads the properties from the device.
```

File: source/_integrations/insteon.markdown [hunk 13]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+<p class='note warning'>If you choose to use the Insteon app, it is recommended to add devices and scenes using the Insteon app. Home Assistant will see the devices and scenes as well. Devices and scenes added in Home Assistant will not be available in the Insteon app.</p>
+<p class='note warning'>Editing a device's All-Link Database can cause the device to become unresponsive. If this occurs, simply relink the device to the modem using the <a href="#add-device">Add device</a> directions above.</p>
 
 [understanding linking]: https://www.insteon.com/support-knowledgebase/2015/1/28/understanding-linking
 
-## INSTEON Scenes
+### Utilities
+
+- **Change the modem connection**: Reconfigure the modem connection information such as USB port or Hub IP address.
+- **Configure device overrides**: Add or remove device overrides. See [Device Overrides](#device-overrides) below.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
- **Configure device overrides**: Add or remove device overrides. See [Device overrides](#device-overrides) below.
```

File: source/_integrations/insteon.markdown [hunk 14]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+
+- **Read the ALDB**: Use the **Load from device** menu option to read the current ALDB records from the device.
+- **Add a record**: Use the **Create ALDB record** menu option to add a record to the All-Link Database. This does not write to the device.
+- **Modify a record**: To modify an existing record, select the record and use the dialog box to change the record as needed. This does not write to the device.
+- **Add default links**: There are a set of default records that need to exist in the ALDB in order for a device to communicate correctly to the modem. Use the **Add default links** menu option to add these links to the device. This **does** write to the device.
+- **Write changes to the device**: Use the **Write changes** menu option to write added or changed records to the device.
+- **Undo changes**: ALDB record changes can be undone prior to being written to the device using the **Undo changes** menu option.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
- **Undo changes**: Undoes ALDB record changes before they are written to the device.
```

File: source/_integrations/insteon.markdown [hunk 15]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+
+### Device All-Link Database
+
+The Insteon All-Link Database (ALDB) contains the list of links to other devices in the Insteon network. The All-Link Database can be managed using the Insteon configuration panel. To see the All-Link Database of a device, select the device from the list of devices in the Insteon configuration panel and select the **All-Link Database** tab.
+
+- **Read the ALDB**: Use the **Load from device** menu option to read the current ALDB records from the device.
+- **Add a record**: Use the **Create ALDB record** menu option to add a record to the All-Link Database. This does not write to the device.
+- **Modify a record**: To modify an existing record, select the record and use the dialog box to change the record as needed. This does not write to the device.
+- **Add default links**: There are a set of default records that need to exist in the ALDB in order for a device to communicate correctly to the modem. Use the **Add default links** menu option to add these links to the device. This **does** write to the device.
+- **Write changes to the device**: Use the **Write changes** menu option to write added or changed records to the device.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]:
```suggestion
- **Write changes to the device**: Writes added or changed records to the device.
```

File: source/_integrations/insteon.markdown [hunk 16]
```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
...
+- **Change device properties**: Select the specific property from the list of properties. Use the dialog box to edit the property value. This does not write the change to the device.
+- **Write property changes**: Use the **Write to device** menu option to save the property changes to the device.
+- **Undo changes**: Property changes can be undone before they are written to the device by selecting the **Undo changes** menu option.
+- **Delete device**: Deletes the Insteon device from Home Assistant and removes all references to the device in the modem. Optionally, it can remove any references to the device in other Insteon devices.
+
+### Device All-Link Database
+
+The Insteon All-Link Database (ALDB) contains the list of links to other devices in the Insteon network. The All-Link Database can be managed using the Insteon configuration panel. To see the All-Link Database of a device, select the device from the list of devices in the Insteon configuration panel and select the **All-Link Database** tab.
+
+- **Read the ALDB**: Use the **Load from device** menu option to read the current ALDB records from the device.
```
Comment (c0ffeeca7) [inline source/_integrations/insteon.markdown]: **Read the ALDB**: is this a UI string?

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Some minor rephrase suggestions

File: source/_integrations/insteon.markdown (hunk 16 above)
Comment (teharris1) [reply source/_integrations/insteon.markdown]: "Load from device" is the UI string which results in reading the All-Link Database (ALDB)

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @teharris1 :+1:

**Pull request merged**